incubator/patroni/values.yaml

```diff
@@ -17,6 +17,11 @@ credentials:
 
 # Distribution Configuration stores
 # Please note that only one of the following stores should be enabled.
+kubernetes:
```

Comment:
IMHO make sense do kubernetes DCS enable by default and save launch a etcd cluster

Reply:
Will do!